### ISSUE TITLE
No motor output detected in output tab

### ISSUE THREAD

**Issue by gabrielvachon** [iNavFlight/inav-configurator#2097]
I have a 5" quad with a Speedybee F7 V3 stack on which i flashed INAV 7.1.1. Everything seems to work fine, however, the outputs tab shows no motor slider (only master) for testing and the ESC doesn't seem to receive any signal from the FC because there is no second sound after I plug the battery. I then tried to flash INAV 6.1.0 and the motors are displayed and the ESC makes a sound to tell that it receives a signal.

<img width="1280" alt="image" src="https://github.com/iNavFlight/inav-configurator/assets/49523085/e04b6d64-4cf9-40ad-8290-0759d560421a">

Steps to reproduce : 
- Flash a SpeedyBee F7 V3 stack with INAV 7.1.1
- Set the model to 5"
- Go to the outputs tab

In conclusion, i can use INAV 6.1.0 but not INAV 7.1.1 on my FC. Is there a known way to fix this?

**Comment by KrackJack1** [comment]
i got the same issue but solved now in mixer tab press load and apply on quad X
hope this help.thx

![Screenshot 2024-05-25 172706](https://github.com/iNavFlight/inav-configurator/assets/28574788/d27ad2c8-0628-422b-b8be-870a7c4ead4e)

**Comment by gabrielvachon** [comment]
It worked for me, thanks a lot!
